web/src/app/users/UserDetails.tsx

```diff
@@ -148,6 +148,36 @@ export default function UserDetails(props: {
     })
   }
 
+  const options: (
+    | JSX.Element
+    | {
+        label: string
+        icon: JSX.Element
+        handleOnClick: () => void
+      }
+  )[] = [
+    <QuerySetFavoriteButton
+      key='secondary-action-favorite'
+      id={userID}
+      type='user'
+    />,
+  ]
+
+  if (isAdmin || userID === currentUserID) {
+    options.unshift({
+      label: 'Edit',
+      icon: <EditIcon />,
```

Comment:
We can change this to the Key icon since it is about access.

Reply:
A cogwheel would be the most familiar icon imo

Reply:
Wouldn't that be general settings rather than passwords/authentication?

Reply:
I'm okay with the key icon in this case as it aligns more closely to the function of the edit modal. The only two functions of this modal as of now are to reset passwords and change admin status, with the former of which being what I assume will have the more prominent usage (esp as only admins can access changing the status). If we add any more options to the modal then I think it would be worth considering a more general icon.

Reply:
Yeah that makes sense since we aren't editing a ton that you didn't use that icon, but we should probably keep the key icon reserved for services. I'd be open to other options.

Reply:
Thoughts on the open lock icon?
<img width="161" alt="Screenshot 2023-05-24 at 9 50 32 AM" src="https://github.com/target/goalert/assets/59105963/6ebea0f4-86ed-4e94-830a-f61c97627076">